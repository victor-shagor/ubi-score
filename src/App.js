import React from "react";
import Signup from "pages/Signup";
import ThankYou from "pages/ThankYou";
import { BrowserRouter as Router, Route, Routes } from "react-router-dom";

function App() {
  return (
    <Router>
      <Routes>
        <Route path="/" element={<Signup />} />
        <Route path="/thankyou" element={<ThankYou />} />
      </Routes>
    </Router>
  );
}

export default App;
