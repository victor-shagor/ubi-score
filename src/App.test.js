import { shallow } from "enzyme";
import App from "./App";
import toJson from "enzyme-to-json";

it("renders without crashing", () => {
  shallow(<App />);
});

it("renders correctly", () => {
  const tree = shallow(<App />);
  expect(toJson(tree)).toMatchSnapshot();
});
