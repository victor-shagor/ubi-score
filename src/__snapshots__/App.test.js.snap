// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<BrowserRouter>
  <Routes>
    <Route
      element={<Signup />}
      path="/"
    />
    <Route
      element={<ThankYou />}
      path="/thankyou"
    />
  </Routes>
</BrowserRouter>
`;
